samples/CommunityToolkit.Maui.Sample/Pages/Views/MediaElement/PositionSlider.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace CommunityToolkit.Maui.Sample.Pages.Views;
+
+/// <summary>
+/// Added a custom slider as a workaround for a bug in the regular Slider. See: https://github.com/dotnet/maui/issues/12285
+/// Original implementation by David Britch: https://github.com/davidbritch/dotnet-maui-videoplayer/blob/main/src/VideoDemos/Controls/PositionSlider.cs
+/// </summary>
+public class PositionSlider : Slider
```

Comment:
I believe we had a TimeSpanTo Double converter in XCT! Can't we just use it here? because we used that converter with MCT's media element in our [project](https://github.com/mhrastegari/fx-files/blob/main/src/Client/App/Views/NativeVideoViewer.xaml#L71) and it worked perfectly with the regular Slider and there was no need to create a custom control and complexing stuff.

Reply:
Yeah I need to look into this again. I still wasn't really happy about it. Still didn't work 100% as intended.